Translation corrections (#348)
07th-mod/higurashi-console-arcs: Update/omot_002_choice1.txt

//r相手は詩音だぞ？　それを良く考えて想像してみろ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "相手は詩音だぞ？　それを良く考えて想像してみろ。",

> 			NULL, "Would Shion ever react that way? Let's put a bit more effort into that simulation.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Would Shion ever react that way? Let's think a little harder about this.", Line_Normal);
```

//rとはいえ、見捨てては寝覚めが悪い…仕方ない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "とはいえ、見捨てては寝覚めが悪い…仕方ない。",

> 			NULL, "Though that said, I'd feel guilty about leaving her to die... there's no other choice.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Though that said, I'd feel guilty about abandoning her... there's no other choice.", Line_Normal);
```
